test/integration/action_execution.go

```diff
@@ -187,6 +190,28 @@ var _ = Describe("Action Executor ", func() {
 		})
 	})
 
+	// Test resize action execution
+	Describe("executing resize action on a deployment", func() {
+		It("should match the expected resource cpu reuqest after resize", func() {
```

Comment:
Lets add another resize action test to test a merged resize action on multiple container specs

Reply:
Regarding this, I'm going to add a new sub task in the current story, and will address it in another PR, @irfanurrehman , what do you think?

Reply:
Sure, that works too.

Reply:
Add a new subtask to trace this: https://vmturbo.atlassian.net/browse/OM-84505